webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -53,8 +54,12 @@ def is_in_s3(s3_client, bucket, crash_id):
 
 def is_in_elasticsearch(supersearch, crash_id):
     """Is the processed crash in Elasticsearch."""
+    crash_date = date_from_ooid(crash_id)
+    start_date = crash_date.strftime("%Y-%m-%d")
+    end_date = (crash_date + datetime.timedelta(days=1)).strftime("%Y-%m-%d")
     params = {
         "uuid": crash_id,
+        "date": [">=%s" % start_date, "<=%s" % end_date],
         "_results_number": 1,
         "_columns": ["uuid"],
         "_facets": [],
```

Comment:
This gives us ES queries like this where the date range is based on the date portion of the crash id.

```
{'query': {'filtered': {'query': {'match_all': {}}, 'filter': {'bool': {'must': [{'bool': {'must': [{'range': {'processed_crash.date_processed': {'lte': '2019-12-14T00:00:00+00:00'}}}, {'range': {'processed_crash.date_processed': {'gte': '2019-12-13T00:00:00+00:00'}}}]}}, {'term': {'processed_crash.uuid': '0000ddbd-1438-4ad4-8d0f-be8992191213'}}]}}}}, 'from': 0, 'size': 1, 'fields': ['processed_crash.uuid']}
```